client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -43,6 +48,7 @@ const CancelJetpackForm: React.FC< Props > = ( {
 	flowType,
 	...props
 } ) => {
+	const provideCancellationOffer = config.isEnabled( 'cancellation-offers' );
```

Comment:
we could think about naming this variable to show more explicitly it's a boolean state; e.g. `shouldProvideCancellationOffer`, `isCancellationOfferEnabled` etc.